### PR TITLE
 [ticket/15987] Table parameters as array

### PR DESCRIPTION
**Pull request by marc1706**
Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15987

### CODE REVIEW

File: phpBB/phpbb/di/extension/tables.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	 *
+	 * @param array            $configs   An array of configuration values
+	 * @param ContainerBuilder $container A ContainerBuilder instance
+	 *
+	 * @throws \InvalidArgumentException When provided tag is not defined in this extension
+	 */
+	public function load(array $configs, ContainerBuilder $container)
+	{
+		if (!$container->hasParameter('tables'))
+			return;
```
Comment (mrgoldy) [inline phpBB/phpbb/di/extension/tables.php]: No brackets here.

Comment (marc1706) [reply phpBB/phpbb/di/extension/tables.php]: sry, used to bad code at work :P

Comment (marc1706) [reply phpBB/phpbb/di/extension/tables.php]: Though this is pretty much the only place where I think this is valid thing to do.

Comment (mrgoldy) [reply phpBB/phpbb/di/extension/tables.php]: Thats fine then aswell, perhaps comment it stating that.

**Pull request merged**